catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -168,18 +166,20 @@ public void configure() throws Exception {
                 }
                 LOGGER.debug("inbox = {}", inbox);
 
-                RouteDefinition routeDefinition = from(inbox);
+                StringBuilder attributeOverrideString = new StringBuilder();
```

Comment:
I don't see an update to the metatatype - I think we want these to be called "attribute overrides" in the directory monitor config instead of parameters